resources/dev-infrastructure.yaml

```diff
@@ -0,0 +1,40 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: Development environment infrastructure for AWS Node Termination Handler.
```

Comment:
Update Descriptions with resources, ex: `Controller and Webhook ECR repos for testing development builds`